.github/workflows/ci-windows-installed.yml

```diff
@@ -28,6 +30,17 @@ jobs:
         codec-aom: 'SYSTEM'
         codec-dav1d: 'SYSTEM'
         codec-rav1e: 'SYSTEM'
+        extra-cache-key: ${{ matrix.compiler }}
+    - name: Leave compiler as default
+      if: matrix.compiler == 'msvc'
+      run: |
+        echo "AVIF_CMAKE_C_COMPILER=" >> $env:GITHUB_ENV
+        echo "AVIF_CMAKE_CXX_COMPILER=" >> $env:GITHUB_ENV
```

Comment:
Note: We can deal with this later.

This generates two cmake warnings:

```
CMake Warning:
  Ignoring empty string ("") provided on the command line.


CMake Warning:
  Ignoring empty string ("") provided on the command line.
```

See the log file at https://github.com/AOMediaCodec/libavif/actions/runs/9976134198/job/27567560349?pr=2270.

Reply:
I guess we can do nothing and the CMake instruction will simply convert non-existent variables.

Reply:
There are other warnings in the log file.

This one should be easy to fix:

```
  Manually-specified variables were not used by the project:

    AVIF_ENABLE_EXPERIMENTAL_AVIR
```

@y-guyon Yannis, Is that the old name for `AVIF_ENABLE_EXPERIMENTAL_METAV1`?

This one I don't understand:

```
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
Hint: The project() command has not yet been called.  It sets up system-specific search paths.
```

Reply:
For the last warning, it's because we do `include(FindPkgConfig)` before calling `project()` in CMake.

Reply:
Thanks. I will write a pull request to fix that.